google-ads-examples/src/main/java/com/google/ads/googleads/examples/accountmanagement/GetAccountHierarchy.java

```diff
@@ -18,50 +18,37 @@
 import com.google.ads.googleads.examples.utils.ArgumentNames;
 import com.google.ads.googleads.examples.utils.CodeSampleParams;
 import com.google.ads.googleads.lib.GoogleAdsClient;
-import com.google.ads.googleads.v3.services.CustomerServiceClient;
 import com.google.ads.googleads.v3.errors.GoogleAdsError;
 import com.google.ads.googleads.v3.errors.GoogleAdsException;
 import com.google.ads.googleads.v3.resources.CustomerClient;
 import com.google.ads.googleads.v3.services.GoogleAdsRow;
 import com.google.ads.googleads.v3.services.GoogleAdsServiceClient;
 import com.google.ads.googleads.v3.services.GoogleAdsServiceClient.SearchPagedResponse;
-import com.google.ads.googleads.v3.services.ListAccessibleCustomersRequest;
-import com.google.ads.googleads.v3.services.ListAccessibleCustomersResponse;
 import com.google.ads.googleads.v3.services.SearchGoogleAdsRequest;
 import com.google.common.base.Strings;
-
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.Multimap;
 import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.HashSet;
+import java.util.Set;
 
-/**
- * Gets the account hierarchy of the specified manager account. If you don't specify manager
- * customer ID, the example will instead print the hierarchies of all accessible customer accounts
- * for your authenticated Google account. Note that if the list of accessible customers for your
- * authenticated Google account includes accounts within the same hierarchy, this example will
- * retrieve and print the overlapping portions of the hierarchy for each accessible customer.
- */
+/** Gets the account hierarchy of the specified manager account. */
 public class GetAccountHierarchy {
 
-  private static final int PAGE_SIZE = 1_000;
-
   private static class GetAccountHierarchyParams extends CodeSampleParams {
 
-    @Parameter(names = ArgumentNames.MANAGER_ID)
+    @Parameter(names = ArgumentNames.MANAGER_ID, required = true)
```

Comment:
Could you ensure we agree on changing this behavior first?
It was suggested by two reviewers when creating PHP golden examples, and it was implemented already in Perl.
This will make this library diverge from other libraries.

Reply:
Change reverted.